app/controllers/application_controller.rb

```diff
@@ -10,6 +10,11 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.for(:sign_up) << :type
   end
 
+  def after_sign_in_path_for(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.